scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamWriter.scala

```diff
@@ -48,7 +75,16 @@ class BeamWriter(val beamMode: BeamMode) extends Writer {
               }
```

Comment:
just to be clear: the above isn't blocking when it calls `run` or `write` is it? It kind of looks like it. I think we need a comment.